sdcm/cluster_gce.py

```diff
@@ -319,6 +319,12 @@ def set_tags_as_labels(_instance):
                 sudo systemctl disable sshguard
                 sudo systemctl stop sshguard
             """)
+
+        if self.params.get("scylla_linux_distro") in ("ubuntu-jammy"):
```

Comment:
Can you check if this has an effect on other distros ?
As far as I know, those parameters aren't new.

Maybe we could just set it across the board for all ?

Reply:
Also it might be safer to use the distrio.UBUNTU22

Reply:
I decided to remove this and add the specific script lines to `get_startup_script` in `TestConfig`, as the cluster logic is already confusing as it is.